### PR TITLE
use :recon_map for trimming output to selected fields

### PR DESCRIPTION
**Pull request by feng19**

### CODE REVIEW

File: lib/limit_formatter.ex [hunk 1]
```diff
@@ -0,0 +1,62 @@
+defmodule Extrace.LimitFormatter do
+  @moduledoc false
+  import Inspect.Algebra
+  alias Code.Identifier
+
+  def limit_inspect(term, opts) when is_map(term) do
```
Comment (redink) [inline lib/limit_formatter.ex:12]: Could you please add some docs for this function?

File: lib/limit_formatter.ex [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        case process_map(term) do
+          {_, term} ->
+            Inspect.Any.inspect(term, Identifier.inspect_as_atom(module), opts)
+
+          _ ->
+            Inspect.Any.inspect(term, opts)
+        end
+
+      _ ->
+        process_map(term)
```
Comment (redink) [inline lib/limit_formatter.ex]:
```suggestion
        term
        |> process_map()
```

File: lib/limit_formatter.ex [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        |> inspect_limited_map(opts)
+    end
+  end
+
+  def limit_inspect(term, opts) do
+    Inspect.inspect(term, opts)
+  end
+
+  def process_map(old_term) do
```
Comment (redink) [inline lib/limit_formatter.ex]:
```suggestion
  @doc false
  defp process_map(old_term) do
```

**Comment by feng19** [comment]
Thanks, @redink
I commit a revised version, please check it again.

**Review by redink — APPROVED** [review]
LGTM, thanks for contribution @feng19

**Comment by redink** [comment]
Published new hex version, thanks. @feng19

**Pull request merged**